swagger.json

```diff
@@ -126,6 +126,36 @@
             "description": "This resource could be updated at this time by TACO."
           }
         }
+      },
+      "delete": {
+        "summary": "Delete a TACO Resource.",
+        "description": "Deletes a TACO resource (Collection, Digital Repository Object, File metadata object [not binary] or subclass of those).",
+        "operationId": "deleteResource",
+        "produces": ["application/json"],
+        "parameters": [{
+          "name": "ID",
+          "in": "path",
+          "description": "SDR Identifier for the Resource.",
+          "required": true,
+          "type": "string"
+        }],
+        "responses": {
+          "200": {
+            "description": "TACO resource metadata delete.",
+            "schema": {
+              "$ref": "#/definitions/ResourceResponse"
+            }
+          },
+          "400": {
+            "description": "Invalid ID supplied"
+          },
+          "401": {
+            "description": "You are not authorized to delete a resource in TACO."
+          },
+          "500": {
+            "description": "This resource could be updated at this time by TACO."
```

Comment:
"This resource could not be deleted at this time by TACO" or "Your request could not be processed at this time."